hack/common.sh

```diff
@@ -7,7 +7,17 @@ GO111MODULE="on"
 GOPROXY="https://proxy.golang.org"
 GOROOT="${GOROOT:-go env GOROOT}"
 
-IMAGE_BUILD_CMD="${IMAGE_BUILD_CMD:-docker}"
```

Comment:
I think the issue is that `IMAGE_BUILD_CMD` doesn't exist yet, so setting it in the line below isn't being affcting the subsequent `if` stanzas. Rather than removing this like entirely, only remove the `:-docker` portion so that we inherit any environment variable value without giving it a default.

Reply:
@jarrpa  - but I was under the impression we do inherit the environment variable value even with my code - my code addition is only relevant if IMAGE_BUILD_CMD is not already set?

Reply:
Not without explicitly marking it. The line above is what says "we will always define this value, and it will inherit from the env var *if it exists*". Without it, the assignments you make below in the `if` stanzas only apply within the scope of those stanzas, since if the env var isn't set it doesn't exist in the broader scope.

Reply:
OK, pushed.